custom/icds_reports/tasks.py

```diff
@@ -829,7 +835,30 @@ def prepare_excel_reports(config, aggregation_level, include_test, beta, locatio
             )
         else:
             cache_key = create_excel_file(excel_data, data_type, file_format)
-    if indicator not in (AWW_INCENTIVE_REPORT, LS_REPORT_EXPORT):
+    elif indicator == THR_REPORT_EXPORT:
+        config['aggregation_level'] = 5  # this report on all levels shows data (row) per AWC
+        excel_data = TakeHomeRationExport(
+            config=config,
+            loc_level=config['aggregation_level'],
+            show_test=include_test,
+            beta=beta
+        ).get_excel_data(location)
+        export_info = excel_data[1][1]
```

Comment:
might be clearer to just use generate a new timestamp here, since it is supposed to be time of download anyway. no reason it has to match the one in the excel and it will be clearer what is going on

Reply:
hey, I made this because of time stamp in the file name is to be same a time stamp in the export info.